### PR TITLE
explain how to make file_sd_config work.

### PR DESCRIPTION
**Pull request by robey**

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -279,15 +279,27 @@ all defined files are detected via disk watches and applied immediately. Files m
...
+The `<file_sd_config>` block has this format:
 
 ```
-{
-  "targets": [ "<host>", ... ],
-  "labels": {
-    [ "<labelname>": "<labelvalue>", ... ]
-  }
-}
+file_sd_configs:
```
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: You should remove this line to be consistent with the other examples in on this page

File: content/docs/operating/configuration.md [hunk 2]
```diff
@@ -279,15 +279,27 @@ all defined files are detected via disk watches and applied immediately. Files m
...
+  - names:
+    - "file1.json"
+    - "/etc/prometheus/file2.json"
```
Comment (grobie) [inline content/docs/operating/configuration.md]: Maybe change to `/absolute/path/to/file2.json`?

Comment (fabxc) [inline content/docs/operating/configuration.md]: A directory example would be nice too. `tgroups/*.json` is the allowed format and pointing at single files is not the most scalable/intended approach.

**Comment by brian-brazil** [comment]
Superseded by #257

**Pull request closed without merging**